web/public/css/footer.less

```diff
@@ -29,7 +29,7 @@
   padding-top: 10px;
   float: right;
 }
-.cdf-info {
+.cdf-info .cp-info{
```

Comment:
I think we're missing a comma after .cdf-info, otherwise only cp-info will be 15px and not the cdf-info class